.github/workflows/pytorch112_gpu.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       develop_base_image: nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
       runtime_base_image: nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
       extra_build_args: --build-arg PYTHON_VERSION=PYTHON3.8 --build-arg ENABLE_FIND_FASTEST_APT_SOURCE=OFF
-      extra_envs: -e TORCH_BLADE_BUILD_TENSORRT_STATIC=ON
+      extra_envs: -e TORCH_BLADE_BUILD_TENSORRT_STATIC=OFF
```

Comment:
I'm curious why?

Reply:
Just for product test. Our product better to provide static linked version, to  reduce user's envoiroment request. But we met some compile error for TRT8.5  like `(.text._ZN5torch5blade21GetTorchMlirWhiteListEv+0x17): relocation truncated to fit: R_X86_64_PC32 against .bss._ZGVZN5torch5blade21GetTorchMlirWhiteListEvE10white_list'` in all static link build. This used to occur in torch1.12+cu113 only, so change this to fix this remaining problem. I'm testing some compiler flag locally such as `-fPIC`.